docker-compose.yml

```diff
@@ -52,7 +52,7 @@ services:
     command: [mysqld, --character-set-server=utf8mb4, --collation-server=utf8mb4_unicode_ci, --wait_timeout=28800, --log-warnings=0]
 
   cache:
-    image: redis:4
+    image: redis:6
```

Comment:
No breaking changes?

Reply:
I have tested it without any problem. Furthermore, the library redis-py is developed against redis 6.0.9 now.